addons/payment_stripe/data/payment_proxy_data.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="fallback_proxy_url" model="ir.config_parameter">
+        <field name="key">payment_stripe.fallback_proxy_url</field>
```

Comment:
Why call this `fallback` and why is it configurable? Can people use a different url?

Reply:
As a first intention, the proxy URL is configurable so that we can test the proxy on a custom URL during our tests.

But we should maybe only hard-code the URL in const.py and remove this ICP before merging this PR.
cc @AntoineVDV

Reply:
I made the suggestion to name it "fallback" because there was already an ICP that was meant to be *the* URL of the proxy. After consideration, I'm also starting to think that keeping only the hard-coded URL should be enough, given that we can configure its mapping.